ckanext/canada/schemas/organization.yaml

```
@@ -227,3 +227,26 @@ fields:
     fr: Courriel de la gouvernement ouvert
   validators: scheming_required email_validator
   display_snippet: email_with_parameters.html
+
+# Field = Public Access.
+# Default value = Government of Canada network.
+# {Determines if the organization's access to the Registry is through a public or closed network}
+- field_name: registry_access
+  label:
+    en: "Access to the Registry"
+    fr: "TBD"
+  help_text:
+    en: The organization will be accessing the Registry through a public or internal (Government of Canada) network
+    fr: TBD
+  choices:
+  - label:
+      en: "Public network"
+      fr: "TBD"
+    value: "public"
+  - label:
+      en: "Government of Canada network"
+      fr: "TBD"
+    value: "goc"
+  form_snippet: radio.html
+  display_snippet: select.html
+  validators: protect_registry_access scheming_choices
```

Something like this might work to make it appear that all current orgs are set to `goc`

```suggestion
  validators: protect_registry_access scheming_choices
  output_validators: default(goc)
  required: true
```